main/supervisor/APKBUILD

```diff
@@ -1,14 +1,15 @@
 # Contributor: Peter Bui <pnutzh4x0r@gmail.com>
 # Maintainer:
 pkgname=supervisor
-pkgver=3.3.5
+pkgver=4.0.1
 pkgrel=0
 pkgdesc="A system for controlling process state under UNIX"
 url="http://supervisord.org"
 arch="noarch"
 license="BSD"
```

Comment:
It seems they moved to a "custom" licence:
https://github.com/Supervisor/supervisor/blob/master/COPYRIGHT.txt

Reply:
No idea how to add custom license(